keras_hub/src/models/backbone.py

```diff
@@ -188,11 +194,13 @@ def save_to_preset(self, preset_dir):
 
     def get_lora_target_names(self):
```

Comment:
Let's rename this everywhere to `default_lora_layer_names`. Potentially we should name this a property too, without a setter.